packages/pieces/math/src/lib/actions/multiplication.ts

```diff
@@ -0,0 +1,25 @@
+import { PieceAuth, Property, Validators, createAction } from "@activepieces/pieces-framework";
+
+export const multiplication = createAction({
+    name: 'multiplication_math',
+    auth: PieceAuth.None(),
+    displayName: 'Multiplicattion',
+    description: 'Multiply first number by the second number',
+    props: {
+        first_number: Property.Number({
+            displayName: 'First Number',
+            description: undefined,
+            required: true,
+            validators: [Validators.number]
```

Comment:
You can remove validators: [Validators.number] as it's included by default in Property.Number